package.json

```
@@ -60,7 +60,7 @@
     "release-it": "^12.4.1"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": ">=16.0.0",
+    "react-dom": ">=16.0.0"
```

What if React 18 introduces a breaking change?

---

What if React 18 doesn't introduce a breaking change?

I kid 😅  I know there are differing opinions on this but IMHO, keeping this open is more useful.

Anyone updating their app to a new major release of React is (I would hope) testing for breakages in their app. By creating a constraint ahead of time, we prevent users from making that upgrade until confirmation for this package is made (that confirmation coming in the form of a new release with the dependency constraint updated).

Yarn resolutions can work around this but that only applies to folks using `yarn` and not `npm`.

---

Could this work? Would prevent React 18 from breaking something once it releases.
```suggestion
    "react": "^16.0.0 || ^17.0.0",
    "react-dom": "^16.0.0 || ^17.0.0"
```